handler/command.go

```diff
@@ -42,6 +43,10 @@ func (h *CommandHandler) commands(c echo.Context) error {
 
 	user, err := h.userSvc.Find(s.UserID)
 	if err != nil {
+		if err == gorm.ErrRecordNotFound {
```

Comment:
This should be, `errors.Is(err, gorm.ErrRecordNotFound)` cause the error has been wrapped